ibis/file/csv.py

```diff
@@ -1,45 +1,24 @@
-import numpy as np
+import toolz
 import pandas as pd
 
-import ibis.expr.datatypes as dt
+import ibis.expr.schema as sch
 import ibis.expr.operations as ops
 
 from ibis.file.client import FileClient
 
 from ibis.pandas.api import PandasDialect
 from ibis.pandas.core import pre_execute, execute  # noqa
-from ibis.pandas.client import pandas_dtypes_to_ibis_schema
 from ibis.pandas.execution.selection import physical_tables
 
 
-_IBIS_TO_PANDAS_DTYPE = {
-    # dt.Any: None,
-    # dt.Null: None,
-    dt.Boolean: bool,
+def _read_csv(path, schema, **kwargs):
+    dtypes = dict(schema.to_pandas())
 
-    dt.Int8: np.int8,
-    dt.UInt8: np.uint8,
-    dt.Int16: np.int16,
-    dt.UInt16: np.uint16,
-    dt.Int32: np.int32,
-    dt.UInt32: np.uint32,
-    dt.Int64: np.int64,
-    dt.UInt64: np.uint64,
+    dates = list(toolz.valfilter(lambda s: s == 'datetime64[ns]', dtypes))
+    dtypes = toolz.dissoc(dtypes, *dates)
 
-    dt.Float: np.float32,
-    dt.Double: np.float64,
-    dt.Halffloat: np.float16,
-
-    dt.String: str,
-    dt.Binary: bytes,
-
-    dt.Date: 'datetime64[D]',
-}
-
-
-def ibis_schema_to_pandas_dtypes(schema):
-    items = zip(schema.names, schema.types)
-    return {name: _IBIS_TO_PANDAS_DTYPE[type(t)] for name, t in items}
+    return pd.read_csv(str(path), dtype=dtypes, parse_dates=dates,
+                       encoding='utf-8', **kwargs)
```

Comment:
Currently each backend on each python versions gives the `string_col` as an unicode column - which requires the utf8 encoding in case of CSV.

OTOH Have You seen my name? :)